ui/src/components/Tasks/Characterisation.js

```diff
@@ -15,6 +15,9 @@ import {
   FieldsRow,
   CollapsableRows,
   toFixed,
+  getLastUsedParameters,
```

Comment:
Added a simply functionality to "remember" (store dialog values in local storage) as `redux-persist` was removed. The general plan is to simplify the overall state management and also replace the `redux-form `based dialogs with something simpler and better maintained. So this local storage solution with `getLastUsedParamters` is an intermediate solution.